src/OpenTelemetry.Exporter.Prometheus.HttpListener/OpenTelemetry.Exporter.Prometheus.HttpListener.csproj

```diff
@@ -0,0 +1,39 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>net462</TargetFrameworks>
+    <Description>Prometheus exporter HttpListener for OpenTelemetry .NET</Description>
+    <PackageTags>$(PackageTags);prometheus;metrics</PackageTags>
+    <MinVerTagPrefix>core-</MinVerTagPrefix>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <NoWarn>$(NoWarn),1591</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <NoWarn>$(NoWarn),1702</NoWarn>
+  </PropertyGroup>
```

Comment:
I think separator here is `;` (but not totally sure). You can also combine these two blocks into one like...

```xml
  <PropertyGroup>
    <NoWarn>$(NoWarn);1702;1591</NoWarn>
  </PropertyGroup>
```

Also might be nice to add a comment about why these particular warnings are being suppressed.

Reply:
It turns out that there's really no good reason to suppress these warnings and I've removed the suppression and fixed them properly.